src/logic/graphMapping.ts

```diff
@@ -0,0 +1,115 @@
+
+import { Graph, MappedEdge, MappedNode } from './commonInterfaces';
+
+const LABEL_ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+export class Edge {
+  label?: number;
+  from: number;
+  to: number;
+  equals = (other: Edge) => {
+    return this.label === other.label && (
+      (this.from === other.from && this.to === other.to) ||
+      (this.from === other.to && this.to === other.from)
+    )
+  };
+  constructor(from: number, to: number, label?: number) {
+    this.from = from;
+    this.to = to;
+    this.label = label;
+  }
+};
+
+/**
+ * Finds all edges in graph matrix. This maybe inefficient (complexity of
+ * O(n^3)), but for this purpose it's ok. Tested up to 2016 edges. For
+ * that amount takes < 1s. Maybe a bottleneck in the future.
+ */
+export function getEdgesArray(graph: Graph): Array<Edge> {
+  const edges: Set<Edge> = new Set();
+  const isAccounted = (edge: Edge) => {
```

Comment:
`isAccounted` sound for me like related to financial stuff so let's maybe find other words